packages/cli/src/utils/base-command.js

```diff
@@ -33,6 +33,13 @@ class BaseCommand extends Command {
       char: "v",
       description: "Config version"
     }),
+
+    //refresh token to use in a CI server
+    token: flags.string({
+      char: "T",
```

Comment:
should be lower case, not upper case

Reply:
We already using the lower t in the create command.
``` 
 // path to the template
    template: flags.string({
      char: "t",
      description: "Path to a git repository with the template",
      required: true
    }),
```

Reply:
will check, approved for now